spec/factories/cx_collection_detail.rb

```diff
@@ -6,7 +6,7 @@
 
     trait :with_cx_collection_detail_upload do
       after(:create) do |cx_collection_detail|
-        FactoryBot.create(:cx_collection_detail_upload, cx_collection_detail: cx_collection_detail)
```

Comment:
given the key matches, it doesn't need to passed in.

this is optional, and primarily aesthetic